angular/projects/admin-nrpti/src/app/records/management-plans/management-plan-detail/management-plan-detail.component.ts

```diff
@@ -14,7 +14,7 @@ import { RecordUtils } from '../../utils/record-utils';
 export class ManagementPlanDetailComponent extends RecordComponent implements OnInit, OnDestroy {
   private ngUnsubscribe: Subject<boolean> = new Subject<boolean>();
 
-  constructor(public route: ActivatedRoute, public router: Router, public changeDetectionRef: ChangeDetectorRef) {
+constructor(public route: ActivatedRoute, public router: Router, public changeDetectionRef: ChangeDetectorRef) {
```

Comment:
Not needed.

Reply:
Removed. Unintended change after some refactoring.